### PR TITLE
Notify route with SMS sending functionality

### LINKED ISSUE #34: EXTENSION: Text Capability
- [x] Configure app for text messaging 
- [x] Confirm how to get user phone number and phone number they want to send info to from FE
- [x] Decide what message should contain
** All Twilio needs is to, from, and body. 
** Right now working with trial phone number (can we assign our test app this phone number?)

**Resources**
- [Twilio SMS Python Quickstart](https://www.twilio.com/docs/sms/quickstart/python)
- [Twilio API Client](https://pypi.org/project/twilio/)
- [Twilio Python Helper Library](https://www.twilio.com/docs/libraries/python)
- [send successful test SMS](https://www.twilio.com/docs/iam/test-credentials?code-sample=code-create-a-message&code-language=Python&code-sdk-version=6.x)

### PR DESCRIPTION
**Pull request by freeheeling**
## What functionality does this accomplish?
closes #34

**Description:**
Configure app to send an SMS via the `/notify` endpoint, using a trial Twilio phone number (`from` param), validated recipient phone number (`to` param), and message body (`body` param) that includes an invitation to join the message sender at a Hangry Ateball-recommended restaurant, with restaurant name and address.

## What did you struggle on to complete?
Understanding what's possible with Twilio's free trial phone number.

### Test Coverage Percentage: ?%
- [x] No Tests have been changed
- [ ] New Tests have been implemented
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code (**manually**)
- [ ] I have partially tested my code (please explain why):

## Helpful Resources:
* Resource link AND small description of info gathered/learned
- [Twilio SMS Python Quickstart](https://www.twilio.com/docs/sms/quickstart/python)
- [Twilio API Client](https://pypi.org/project/twilio/)
- [Twilio Python Helper Library](https://www.twilio.com/docs/libraries/python)
- [send successful test SMS](https://www.twilio.com/docs/iam/test-credentials?code-sample=code-create-a-message&code-language=Python&code-sdk-version=6.x)

## Review Requests(optional):
@linda-le1, @Jonpatt92 

## Please include a gif of how you feel about this branch:
alt-text

### CODE REVIEW

File: base_app/base.py [hunk 1]
```diff
@@ -36,10 +39,25 @@ def get_random():
     params = format_params(request.args)
     return service.get_recommendation(params )
 
+@app.route('/api/v1/notify', methods=['GET'])
```
Comment (linda-le1) [inline base_app/base.py:42]: Awesome! I want to make sure I'm understanding this, so let me know if this isn't exactly how it works: So we assign this route to the "Send to Friends" button, FE sends params as 'from', 'to', and 'body', a text is created, and then gets sent. Is 'body' just the restaurant name?

**Review by linda-le1 — APPROVED** [review]
Nice work on implementing the text feature with Twilio!

**Review by kimmccaskill — APPROVED** [review]
Hyped for this!

File: base_app/base.py (hunk 1 above)
Comment (kimmccaskill) [reply base_app/base.py:42]: body is going to be "Brothers Bar at 1900 Market St. Denver, CO"

Comment (kimmccaskill) [reply base_app/base.py:42]: so "(restaurant.name) at (restaurant.location)"

**Pull request merged**